### PR TITLE
document new syslog module syntax

### PR DESCRIPTION
**Pull request by floren**

### CODE REVIEW

File: search/syslog/syslog.md [hunk 1]
```diff
@@ -56,7 +56,13 @@ The syslog module would extract the following fields:
 * MsgID: "1"
 * Message: "An application event log entry..."
 
-The portion `[ex@32473 iut="3" foo="bar"]` is the *Structured Data* section. Structured Data sections contain the structured value ID ("ex@32473", extracted with the `StructuredID` keyword) and any number of key-value pairs. To access a value using the syslog module, specify the key: executing `syslog iut` will set an enumerated value named `iut` containing the value "3". Similarly, `syslog StructuredID foo` would extract `StructuredID` containing "ex@32473" and `foo` containing "bar".
+### Structured Data
+
+In the example record above, the portion `[ex@32473 iut="3" foo="bar"]` is the *Structured Data* section. Structured Data sections contain the structured value ID ("ex@32473", extracted with the `StructuredID` keyword) and any number of key-value pairs. To access a value using the syslog module, specify `Structured.key`: executing `syslog Structured.iut` will set an enumerated value named `iut` containing the value "3". Similarly, `syslog StructuredID Structured.foo` would extract `StructuredID` containing "ex@32473" and `foo` containing "bar".
```
Comment (traetox) [inline search/syslog/syslog.md:61]: executing->specifying

File: search/syslog/syslog.md [hunk 2]
```diff
@@ -56,7 +56,13 @@ The syslog module would extract the following fields:
...
+
+Note that a single syslog message may contain multiple structured data sections, each with its own ID. If both sections define the same key, you may wish to explicitly specify which section to extract from. This can be done by inserting the Structured ID into the extraction: `syslog Structured[ex@32473].foo`. If you do not specify the ID, the module will extract a result from one of the sections with no guarantees as to which one.
+
+Note: If multiple structured data sections exist, extracting the StructuredID field will return the ID of the first section. Filters will be checked against the IDs of all sections. For example, given an entry containing `[foo@bar a=b][baz@quux x=y]`, the entry will be dropped if you specify `StructuredID!="baz@quux"` *or* `StructuredID!="foo@bar"`.
```
Comment (traetox) [inline search/syslog/syslog.md:65]: add another example with multiple:

Alternatively, specifying "StructuredID==baz@quux" will only drop entries that do not contain any structured data sections without "baz@quux".

**Review by traetox — CHANGES_REQUESTED** [review]
nitpicks

**Comment by floren** [comment]
PTAL

**Comment by traetox** [comment]
LTGM

**Review by traetox — APPROVED** [review]
LGTM

**Pull request merged**